etc/xbps.d/repos-local-x86_64.conf

```diff
@@ -1,3 +1,4 @@
-# Local repositories
+# DON'T EDIT THIS FILE
```

Comment:
Any time there's a "Don't edit this file" comment its always nice to have a pointer to the files you *can* edit.

Reply:
the `repos-local*.conf` files do not need to be modified because xbps-src takes care of changing them in all cases.

Reply:
right, it's unnecessary